sample_files/.travis.yml

```diff
@@ -11,9 +11,21 @@ addons:
 
 env:
   global:
-  - VERSION="8.0" TESTS="0" LINT_CHECK="0"
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
```

Comment:
Not introduced in this PR but: why do we have TESTS="0" on the global env? Doesn't it make more sense to be TESTS="1" and then use TESTS="0" for the build that don't run tests?